### ISSUE TITLE
Change the (natural) language of the Notebook interface back to English

### ISSUE THREAD

**Issue by grumpfou** [jupyter/notebook#4158]
Recently all the interface of the Jupyter notebook has been automatically translated into the my own language (French). It is the case on all the web browsers I've tested (Firefox, Chrome, Chromium). I've tried to bring it back to English. It seems there was no option in the Jupyter-notebook config file; I tried to change the interface of the web browser itself and it had no effect. 

The only solution I found was to open a terminal and write 
```
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
```
before launching jupyter-notebook.

I would had imagine a more convenient way to change the language of the notebook.

Python: 3.6.6
Jupyter-notebbok --version : 5.7.0
OS : Ubuntu 18.04

**Comment by grumpfou** [comment]
@Carreau

**Comment by Carreau** [comment]
Thanks for opening this. 

I'm pretty sure I managed to dynamically switch at some point last time I tried the in internationalization. I see some information in the internationalisation readme, buthttps://github.com/jupyter/notebook/blob/04a686dbaf9dfe553324a03cb9e6f778cf1e3da1/notebook/i18n/README.md but it should at least be better documented on in the rendered documentation on readthedoc.

**Comment by Carreau** [comment]
so I poked around and I'm not sure how to dynamically change language. 

I've found [this](https://github.com/aio-libs/aiohttp-jinja2/issues/187), but that looks heavy handed.

**Comment by bensdm** [comment]
same issue, any solution?

**Comment by yltang52** [comment]
same issue, any solution?

**Comment by remorsecs** [comment]
Ref: https://github.com/jupyter/notebook/tree/5.7.x/notebook/i18n#how-the-language-is-selected-
Linux solution:
Open terminal, and set
``` bash
export LANGUAGE=
```

**Comment by sdlins** [comment]
> Ref: https://github.com/jupyter/notebook/tree/5.7.x/notebook/i18n#how-the-language-is-selected-
> 
> Linux solution:
> Open terminal, and clear the environment variable $LANGUAGE,
> e.g. (bash)
> 
> ```shell
> export LANGUAGE=
> ```
> 
> and then run jupyter notebook on this shell.

Unfortunately, this tip didn't work for me (Linux Deepin). At least not completely because when searching commands via Command Pallete (CTRL + SHIFT + P) the commands list is still in French! :-/

Any workaround?

**Comment by Chinh052010** [comment]
You can fix it by go to Setting in Chrome => Advanced => Languages => put the language you want to use in the top 
![image](https://user-images.githubusercontent.com/32285868/56021770-75599b00-5d34-11e9-9578-1b97d9989838.png)

**Comment by phihh** [comment]
> You can fix it by go to Setting in Chrome => Advanced => Languages => put the language you want to use in the top
> ![image](https://user-images.githubusercontent.com/32285868/56021770-75599b00-5d34-11e9-9578-1b97d9989838.png)

thanks a lot. this works for me in windows 10.(I didn't use powershell to fix this problem)

**Comment by lmingzhi** [comment]
> Recently all the interface of the Jupyter notebook has been automatically translated into the my own language (French). It is the case on all the web browsers I've tested (Firefox, Chrome, Chromium). I've tried to bring it back to English. It seems there was no option in the Jupyter-notebook config file; I tried to change the interface of the web browser itself and it had no effect.
> 
> The only solution I found was to open a terminal and write
> 
> ```
> export LC_ALL=en_US.UTF-8
> export LANG=en_US.UTF-8
> ```


It works for me under :
Python: 3.7.3
jupyer 4.4.0
MacOS 10.14.3

**Comment by dev-techmoe** [comment]
```bash
LANGUAGE="" LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 jupyter notebook
```
this works for me.

**Comment by Lucecpkn** [comment]
> You can fix it by go to Setting in Chrome => Advanced => Languages => put the language you want to use in the top
> ![image](https://user-images.githubusercontent.com/32285868/56021770-75599b00-5d34-11e9-9578-1b97d9989838.png)

This works for me in Win10 chrome. Thank you so much! 
I have always tried to modify the language back to English for webpages in chrome.  This language configuration is the first place i went to, but i found it only changed the chrome fonts language, but not the webpage default display language.
Until today, i find it's the order of language list that decides the display language when you open a webpage... and we have to move the English on top of the list!

**Comment by buiducsinh34** [comment]
> You can fix it by go to Setting in Chrome => Advanced => Languages => put the language you want to use in the top
> ![image](https://user-images.githubusercontent.com/32285868/56021770-75599b00-5d34-11e9-9578-1b97d9989838.png)

Thanks a lot, problem solved.

**Comment by Mahmoodsalah** [comment]
nothing from solutions above worked with me. 

OS: windows 10

**Comment by HouariZegai** [comment]
There are no solutions mentioned above worked with me!
OS: windows 7

**Comment by MagedSaeed** [comment]
Solutions above did not work for me!
OS: Windows 10. Built 1903

**Comment by yassinose00** [comment]
> You can fix it by go to Setting in Chrome => Advanced => Languages => put the language you want to use in the top
> ![image](https://user-images.githubusercontent.com/32285868/56021770-75599b00-5d34-11e9-9578-1b97d9989838.png)

It worked like charm, thanks.

**Comment by Vercaca** [comment]
> nothing from solutions above worked with me.
> 
> OS: windows 10

```
set LC_ALL=en_US.UTF-8
set LANG=en_US.UTF-8
```

It's work!!

**Comment by kenmaxnejp** [comment]
> Ref: https://github.com/jupyter/notebook/tree/5.7.x/notebook/i18n#how-the-language-is-selected-
> 
> Linux solution:
> Open terminal, and clear the environment variable $LANGUAGE,
> e.g. (bash)
> 
> ```shell
> export LANGUAGE=
> ```
> 
> and then run jupyter notebook on this shell.

It works at ubuntu 18.04.

**Comment by HongYang57** [comment]
This is my solution:   /anaconda3/lib/python3.7/site-packages/notebook/i18n/xxx    <---- Try rename this  xxx  file to anything else and reopen jupyter notebook.
e.g.  /anaconda3/lib/python3.7/site-packages/notebook/i18n/zh_CN  ----->  zh_CN_old   and reopen jupyter notebook.

**Comment by klchang** [comment]
> This is my solution: /anaconda3/lib/python3.7/site-packages/notebook/i18n/xxx <---- Try rename this xxx file to anything else and reopen jupyter notebook.
> e.g. /anaconda3/lib/python3.7/site-packages/notebook/i18n/zh_CN -----> zh_CN_old and reopen jupyter notebook.

It works for me. In Windows 7, change X:\xxx\Anaconda3\Lib\site-packages\notebook\i18n\\**zh_CN** to  X:\xxx\Anaconda3\Lib\site-packages\notebook\i18n\\**zh_CN_old**.

**Comment by tandav** [comment]
> You can fix it by go to Setting in Chrome => Advanced => Languages => put the language you want to use in the top
> ![image](https://user-images.githubusercontent.com/32285868/56021770-75599b00-5d34-11e9-9578-1b97d9989838.png)

For me just putting prefered language on top was not enough. I had to remove undesired language from that list. Then it showed in english.

**Comment by roobooot** [comment]
> > You can fix it by go to Setting in Chrome => Advanced => Languages => put the language you want to use in the top
> > ![image](https://user-images.githubusercontent.com/32285868/56021770-75599b00-5d34-11e9-9578-1b97d9989838.png)
> 
> For me just putting prefered language on top was not enough. I had to remove undesired language from that list. Then it showed in english.

Solved by this.

**Comment by 3chdog** [comment]
> ```shell
> LANGUAGE="" LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 jupyter notebook
> ```
> 
> this works for me.

This works for me too. Thank you!

I tried these settings
"
export LC_ALL=en_US.UTF-8
export LANG=en_US.UTF-8
"
but it didn't work for me.

my env:
conda version: 4.10.1
python version: 3.6.13
ubuntu: 20.04

**Comment by ClancyCC** [comment]
> > You can fix it by go to Setting in Chrome => Advanced => Languages => put the language you want to use in the top
> > ![image](https://user-images.githubusercontent.com/32285868/56021770-75599b00-5d34-11e9-9578-1b97d9989838.png)
> 
> For me just putting prefered language on top was not enough. I had to remove undesired language from that list. Then it showed in english.

Thank u sooooooo much! It works for me!!!!

**Comment by laowantong** [comment]
Under macOS 12.5.1 / Chrome 105.0, I needed to combine three of the above workarounds:

1. Under Chrome Preferences, put English on top.
2. Under Chrome Preferences, suppress my local language.
3. Under terminal, language voodoo before launching the command.

I defined a shortcut `juno` in my `.bash_profile`:

```shell
alias juno='LANGUAGE="" LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 jupyter notebook --browser chrome'
```

**Comment by rijieli** [comment]
> This is my solution: /anaconda3/lib/python3.7/site-packages/notebook/i18n/xxx <---- Try rename this xxx file to anything else and reopen jupyter notebook.
> e.g. /anaconda3/lib/python3.7/site-packages/notebook/i18n/zh_CN -----> zh_CN_old and reopen jupyter notebook.

That's really what I want! I don't want to change ENV language settings.

**Comment by jtpio** [comment]
Closing as this will be fixed in the upcoming `7.0` release:

https://github.com/jupyter/notebook/assets/591645/6e90849c-58e4-4b25-90bf-b6357d232860

You can try the Notebook 7 pre-release with `pip install --pre notebook`.

Feel free to open a new topic on [Discourse](https://discourse.jupyter.org/) if you have more questions, thanks!
